.bazelversion

```diff
@@ -0,0 +1 @@
+6.4.0
```

Comment:
I noticed that CI was using 6.4.0 so I mimic that here. We can drop this, or land this separately... or keep this here?

Reply:
This is fine, we can and should bump it separately.

Reply:
Now I remember why I added this. The repo doesn't yet support bazel 7.x.x, so I mimic'd what CI is using locally to ensure tests pass locally.

The failures:

* `//cmd/gazelle:gazelle_test` fails with: `could not locate go tool`
* `//internal:bazel_test` fails with: `error: unknown runfile: ./.bazelrc`

Reply:
I could go and fix these issues, but I suspect that should be a seperate PR (I don't think it's related to this one) But let me know.